build.sbt

```diff
@@ -121,12 +122,20 @@ lazy val http2Support = project("akka-http2-support")
       resourceGenerators in Test += Def.task {
         val log = streams.value.log
         val h2spec = h2specPath.value
-
         if (!h2spec.exists) {
           log.info("Extracting h2spec to " + h2spec)
 
-          for (zip <- (update in Test).value.select(artifact = artifactFilter(name = h2specName, extension = "zip")))
-            IO.unzip(zip, (target in Test).value)
+          for (archive <- (update in Test).value.select(artifact = artifactFilter(name = h2specName, extension = h2specExt))) {
```

Comment:
Could you add the version number to the path, so that it is redownloaded when we change the version number? I guess we can do that by changing the path like this:

```scala
     lazy val h2specPath = Def.task {
       (target in Test).value / h2specName / Dependencies.h2specVersion / h2specExe
     }
```

Can you try if that works?